governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
   owners shouldn't scale with the organization's growth, O(1), and optimally it
   should be less than 20 people who are very familiar with project workings and
   distributed across a few time zones and organizations The other repos will
   have distinct sets of people filling some of the above roles, also.
 
+## Procedural roles
+
+- **Champion**
+  - A Champion is the primary point of contact for guiding someone through a process, such
+    as [creating a new incubator repo](incubator.md) or becoming a maintainer
+  - The majority of the mentorship, review, and advice regarding Kubernetes community norms and
+    processes will come from the Champion
+  - Potential Champions come from a group of existing Kubernetes contributors, such as REVIEWERS
+    or MAINTAINERS -- which group depends on the particular process 
+- **Sponsor**
+  - A Sponsor is an approver for initiating a process, such as creating a new incubator repo or
+    adding a new maintainer
+  - Potential Sponsors come from a very small set of senior Kubernetes contributors (typically LEADS)
+  - The idea is that by relying on this small set of Kubernetes Community members to approve
+    will ensure consistency and preserve the culture and integrity of the project.
+  - Being a Sponsor is a minor advisory role
+
 ## Other repositories
 
-Guidelines for roles in other repositories are TBD. New subprojects/repositories need to be
-able to add reviewers, approvers, and maintainers more rapidly than more mature subprojects.
-Subprojects less than 1 year old will have relaxed time and PR requirements.
+Guidelines for roles in other repositories are TBD, but some known special cases are called out below. 
+
+Notable repositories/categories:
+
+- `kubernetes.github.io`: In addition to the typical repo-oriented roles, `kubernetes-pm` has write
+  permission to this repo in order to manage issues (labels, milestones, etc.), and
+  `kubernetes-maintainers` and `kubernetes-reviewers` have write and read permissions, as well
+- `features`: In addition to the typical repo-oriented roles, `kubernetes-pm` has write permission
+  to this repo in order to manage issues (labels, milestones, etc.) and the issue template, and
+  `kubernetes-maintainers` and `kubernetes-reviewers` have write and read permissions, as well
+- `community`: This repository is intended to be fairly open to the community. `kubernetes-maintainers`,
+  `kubernetes-pm`, SIG-related github teams, and other trusted groups may be given write access in
+  order to help maintain SIG and contributor documentation
+- `test-infra`: In addition to the typical repo-oriented roles, `kubernetes-build-cops` have admin
+  access to this repository in order to fix CI problems.
+- `release`: In addition to the typical repo-oriented roles, `kubernetes-build-cops` have admin access
```

Comment:
Specifically the stuff in `kubernetes/release` cannot be run by non-googlers.  The gcr repo that is used conflates GCP images with kubernetes images.  non-googlers have not been (can not be?) added to the ACLs for GCS buckets.  The deb/rpm repos are also google only and pushed with scripts and mechanisms that are not part of any k8s repo.

This is different from a non-Google release manager -- that is a herding the cats role.

And while I 100% believe that this was all done out of expediency and not a bid to control the project, the impact is the same.  Non-googlers cannot play a role here.  It will take effort for *google* to untangle this (who else can?!?) but is not a priority.